src/bots/ama/subCommands/deleteEvent.js

```diff
@@ -0,0 +1,36 @@
+import EventModel from '../models/Event';
+import client from '../../../client';
+import { getStrings } from '../constants';
+
+/**
+ * Handles deleting an event with Event schema
+ * @param {Array.<string>} args - rest of command arguments
+ */
+const handler = async (args) => {
+  if (args.length != 1) {
+    client.message.channel.send(getStrings().insufficientArgumentsEvent);
+    return;
+  }
+
+  let deleted;
+
+  try {
+    deleted = await EventModel.findOneAndDelete({ _id: args[0] });
+  } catch (_) {
```

Comment:
you could just not pass an argument instead of passing `_`, just do `catch { ... }`